.github/workflows/ci.yml

```diff
@@ -1,20 +1,44 @@
-name: CoralMicro CI Runner
+name: CoralMicro Build
 
 on: [push, pull_request]
 
-env:
-  BUILD_TYPE: Release
-
 jobs:
 
-  build-linux:
+  build-arduino-core:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 20
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 1
+          ssh-key: ${{ secrets.ACTIONS_DEPLOY_KEY }}
+          submodules: recursive
+
+      - name: Setup Environment
+        run: |
+          bash setup.sh
+          python -m pip install ninja
```

Comment:
You use python here, but python3 everywhere else. Let's make them all match.

Reply:
done!